packages/typography/src/Description/Description.types.ts

```diff
@@ -0,0 +1,6 @@
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+export type DescriptionProps = HTMLElementProps<'p', never> & {
+  darkMode?: boolean;
```

Comment:
Why aren't we extending CommonTypographyProps here?

Reply:
I didn't modify implementation details just moved code around.

Reply:
updated